docs/Clojure.md

```diff
@@ -0,0 +1,84 @@
+# How to use Minio SDK for java with Clojure
+
+In this receipe we will learn how to use `minio-java` with Clojure.
+
+## Prerequisites
+Install [`Clojure`](https://clojure.org/community/downloads) and [`Leiningen`](https://leiningen.org/).
+
+Check by running -
+```
+lein --help
+```
+
+## Clojure Project
+Following are the steps involved foe using minio-java with Clojure : 
+
+### 1. Setup
+Create a new Clojure project using Leiningen.
+```
+lein new <templateName> <projectName>
+```
+
+```
+lein new app clojproject
+```
+
+Navigate to `project.clj` look at `:dependencies` section, listing what project requires in order to load, compile and run.
+```
```

Comment:
This line is not clear - please elaborate on what is `dependencies` section and how is `listing` happening